pkg/client/client.go

```diff
@@ -292,13 +297,15 @@ func (client *DataSetClient) listenAndSendBufferForSession(session string, ch ch
 			client.lastAcceptedAt.Store(time.Now().UnixNano())
 			continue
 		}
-		client.Logger.Debug("Received Buffer from channel",
-			zap.String("session", session),
-			zap.Int("processedMsgCnt", processedMsgCnt),
-			zap.Uint64("buffersEnqueued", client.statistics.BuffersEnqueued()),
-			zap.Uint64("buffersProcessed", client.statistics.BuffersProcessed()),
-			zap.Uint64("buffersDropped", client.statistics.BuffersDropped()),
-		)
+		if processedMsgCnt%100 == 0 {
```

Comment:
Should we make this value "log every X messages" a constant / configurable?

Reply:
This message was just flooding the logs. I was initially thinking that I should delete it.

Can we discuss it - how much configurable this should be - and introduce it in some future versions?